### DISCUSSION TITLE
检测到本地服务不是 HTTP/HTTPS 服务器, 自动 HTTPS 已被禁用

### DISCUSSION THREAD

**Discussion by Leonard-Li777** [natfrp/wiki#518] [使用问题] [answered]
之前都好好的，突然提示：Tunnel/myai_cd [64/1833269/d585] 检测到本地服务不是 HTTP/HTTPS 服务器, 自动 HTTPS 已被禁用
实际是http的API服务，是可以访问的。

然后用一个web站点测试，又可以启动SakuraFrp穿透服务

于是我把API服务的根路径get method  直接返回html内容，结果还是不行

windows 2022

**Comment by q32757468** [comment]
遇到相同的问题

**Comment by LTHPKBTE** [comment, marked as answer]
如果您清楚您在做什么，可以通过将 auto_https_mode （自动 HTTPS 工作模式）选项配置为 http 无视自动探测。
![图片](https://github.com/user-attachments/assets/2832fdab-80d4-4f89-9c9c-dac04c52e4dd)
此选项可能需要您打开高级用户模式才可见。

**Reply by Zha-Miku** [reply]
遇到了同样的问题，完美解决了
